src/ccnl-core/include/ccnl-prefix.h

```diff
@@ -119,11 +119,12 @@ ccnl_prefix_cmp(struct ccnl_prefix_s *pfx, unsigned char *md,
  * @param[in] c             Content to test the prefix against
  *
  * @return      -1 if no match at all (all modes) or exact match failed
+ * @return      -2 mismatch in expected number of components between prefix and content
+ * @return      -3 computation of digest failed 
  * @return      0 if full match
  * @return      n>0 for matched components
 */
-int
-ccnl_i_prefixof_c(struct ccnl_prefix_s *prefix,
+int ccnl_i_prefixof_c(struct ccnl_prefix_s *prefix,
```

Comment:
I am wondering about this re-alignement here. All other signatures start with the return type and the function name in the below line.

Reply:
We probably should start to talk about a style guide soon.